src/lxc/commands.c

```diff
@@ -89,6 +89,7 @@ static const char *lxc_cmd_str(lxc_cmd_t cmd)
 		[LXC_CMD_GET_CGROUP_CTX]		= "get_cgroup_ctx",
 		[LXC_CMD_GET_CGROUP_FD]			= "get_cgroup_fd",
 		[LXC_CMD_GET_LIMIT_CGROUP_FD]		= "get_limit_cgroup_fd",
+		[LXC_CMD_GET_SYSTEMD_SCOPE]		= "get_systemd_scope",
```

Comment:
I tend to prefer to send file descriptors, not paths nowadays similar to `get_cgroup_fd()` and `get_limit_cgroup_fd()`..

Reply:
We don't want an FD in this case though, we only want the actual systemd scope name, which I *think* is always just the last piece of the cgroup path.  So we could pass an fd along and do an fstat.  That *should* work.